charts/spire/charts/spire-server/templates/NOTES.txt

```diff
@@ -9,11 +9,6 @@ Installed {{ .Chart.Name }}…
 
 Installed {{ include "spire-tornjak.fullname" . }}…
 
-### WARNING ###
-
-Tornjak runs without authentication and is therefore NOT suitable to run in production environments.
-Only use in test environments!
-
```

Comment:
Should we adapt this to still display the warning when authentication is configured off?

Reply:
@kfox1111, let me make these level changes on my pass through Tornjak documentation (#288 )

Reply:
I'm ok with the documentation bits pulled out to a different pr. This one though is about the code, which this file is also. We should either not remove the warning in this pr or fix it with a quick conditional. Here's one that should solve it. (It wont allow me to do it as a suggestion)

```
{{- if eq .Values.tornjak.config.userManagement.issuer "" }}
### WARNING ###

Tornjak is configured to run without authentication and is therefore NOT suitable to run in production environments.
Only use in test environments!
{{- end }}
```